src/javaExercises/stairCase/StairCase.java

```diff
@@ -0,0 +1,22 @@
+package javaExercises.stairCase;
+
+import java.util.Scanner;
+
+public class StairCase {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input desired staircase height:");
+        int n = sc.nextInt();
+
+        for (int i = 1; i <= n; i++) {
+            for (int j = 1; j <= n; j++) {
```

Comment:
you are doing to many iterations without any purpose, try replacing second `for` with a `while` statement

Reply:
Done